### DISCUSSION TITLE
Tomcat upgrade in DTS

### DISCUSSION THREAD

**Discussion by christopher-satterfield** [percussion/percussioncms#1061] [General]
I was given a task to upgrade Tomcat under DTS.  The Tomcat had been upgraded before from v9.0.71 to v9.0.74 using this procedure:

You can download the 9.0.74 version of Tomcat and copy over the bin/ and lib/ folders to the DTS install in sandbox. Let me know if this works:
• Download apache-tomcat-9.0.74.tar.gz from this link:
https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.74/
• Back up the Deployment/Server/bin and Deplyoment/Server/lib folders
• Copy in the contents of the tomcat 9.0.74 bin and lib folders
• Restart the DTS

Can the same procedure be used to go from v9.0.74 to v9.0.80?

**Comment by natechadwick** [comment]
Hi,

I would recommend updating to a current release to get updated versions of the dependencies.  The 8.1.3 release can be found here:  https://github.com/percussion/percussioncms/releases/tag/v8.1.3.6

-n
